spec/unit/rest/Twilio.spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const nock = require('nock');
+var url = require('url');  /* jshint ignore:line */
+
+describe('client', () => {
+  var client;
+  const twilio = require('../../../lib');
+
+  describe('setting the region', () => {
+    it('should use the default region if only edge is defined', () => {
+      client = new twilio('ACXXXXXXXX', 'test-password');
```

Comment:
This can be constructed outside the testing block. Or thrown into a `beforeEach`.